docs/UserGuide.md

```diff
@@ -97,46 +97,50 @@ Expressions without `...` at the end takes the last parameter as the argument wh
-The `FILE_PATH` field can accept both absolute and relative file path from your current directory in HelloFile.
+You can add a nickname to a file. This make it easier to manage only some specific files that you want such as frequently used files, etc.
+
+Please take note of the following:
+* You can add any number of labels to the tag (including 0).
+* The `FILE_PATH` field can accept both absolute and relative file path from your current directory specified in the internal File Explorer of HelloFile.
+* You can tag a file without access permission.
 
 Format: `tag t>TAG_NAME f>FILE_PATH [l>LABEL]...`
 
 ![tag](images/screenshots/tag_command.png)
 
 Examples:
-* `tag t>Users f>C:\Users` (Adds a tag with tag name `Users`,absolute file path `C:\Users`, and no label)
-* `tag t>Users f>C:\Users l>Important` (Adds a tag with tag name `Users`, absolute file path `C:\Users`, and label name `Important`)
-* `tag t>Users f>.\Users` (Adds a tag with tag name `Users`, relative file path `Users`, and no label)
-* `tag t>Users f>.\Users l>folder l>readonly` (Adds a tag with tag name `Users`, relative file path `Users`, and multiple lables
-with label name `folder` and `readonly`)
+* `tag t>Users f>C:\Users` (Adds a tag with tag's nickname `Users` that points to `C:\Users` with no label to the tag list)
+* `tag t>Users f>C:\Users l>Important` (Adds a tag with tag's nickname `Users` that points to`C:\Users` with a label `Important` to the tag list)
+* `tag t>Users f>.\Users` (Adds a tag with tag's nickname `Users` that points to a folder `Users` in the current directory with no label to the tag list)
+* `tag t>Users f>.\Users l>folder l>readonly` (Adds a tag with tag's nickname `Users` that points to a folder `Users` in the current directory with labels `folder` and `readonly`)
 
 ### Displaying information of a tagged file : `show`
 
-Displays the information of the tagged file.<br>
+Displays the information of the specified tagged file.<br>
 You can see some basic information of a tagged file by using this command.
-It will show you the tag's file path and label information.
+It will show you the tag's file path and label information in case of the UI can't fully show them.
 
 Format: `show t>TAG_NAME`
 
 ![show](images/screenshots/show_command.png)
 
 Examples:
-* `show t>my_research` (show you the details of a tag name `my_research`)
-* `show t>file2020` (show you the details of a tag name `file2020`)
+* `show t>my_research` (show you the details of the tag name `my_research`)
+* `show t>file2020` (show you the details of the tag name `file2020`)
 
 ### Accessing a tagged file : `open`
 
-Opens the file specified by the unique tag name.<br>
+Opens the file specified by the unique tag's nickname or label.<br>
 You can open a file using its tag, with the default application in your operating system.
-Additionally, you can open all files with a specific label. This could potentially open many files.
+Additionally, you can open all files with a specific label.
```

Comment:
maybe it is better to say "Additionally, you can open all files with the same label(?)

Reply:
Yeah, I miss that. Should be with the same label.